authorization/src/main/kotlin/com/ritense/authorization/request/DelegateUserEntityAuthorizationRequest.kt

```diff
@@ -22,9 +22,16 @@ class DelegateUserEntityAuthorizationRequest<T>(
     resourceType: Class<T>,
     action: Action<T>,
     override val user: String?,
-    entity: T?, //TODO: Determine if this really should be nullable
+    entities: List<T>?
 ) : EntityAuthorizationRequest<T>(
     resourceType,
     action,
-    entity
-)
\ No newline at end of file
+    entities
+) {
+    constructor(resourceType: Class<T>, action: Action<T>, user: String?, vararg entities: T) : this(
+        resourceType,
+        action,
+        user,
+        if (entities.any { it == null }) null else entities.filterNotNull().toList()
```

Comment:
does this mean that if you provide a list, but 1 entry is null, the entire list is ignored?

Reply:
Its not ignored. A null of an empty list will result in a permission denied.
Because the code below should still result in an access denied:
```
new EntityAuthorizationRequest(
    CamundaProcessDefinition.class,
    Action.deny(),
    null
)
```

Although there is this use-case that is bothersome. What if the taskService returns null:
```
new EntityAuthorizationRequest(
    CamundaTask.class,
    VIEW,
    taskService.findTaskById(taskId)
)
```